README.md

```diff
@@ -124,6 +124,59 @@ If you use it and encounter any problems, feel free to open issues on [github](h
 To run diktat in **only-check** mode use command `$ mvn diktat:check@diktat`.
 To run diktat in **autocorrect** mode use command `$ mvn diktat:fix@diktat`.
 
+Requesting a specific _Maven_ `executionId` on the command line (the trailing
+`diktat` in the above example) may be essential in these cases:
+
+  * In your `pom.xml`, you have multiple executions with different
+    configurations (e. g.: multiple rule sets):
+    <details>
+
+    ```xml
+    <executions>
+
+        <execution>
+            <id>diktat-basic</id>
+            <configuration>
+                <diktatConfigFile>diktat-analysis.yml</diktatConfigFile>
+            </configuration>
+        </execution>
+
+        <execution>
+            <id>diktat-advanced</id>
+            <configuration>
+                <diktatConfigFile>diktat-analysis-advanced.yml</diktatConfigFile>
+            </configuration>
+        </execution>
+
+    </executions>
+    ```
+    </details>
+  * Your YAML file with DiKTat rules has a non-default name and/or resides in a
+    non-default location:
+    <details>
+
+    ```xml
+    <executions>
+        <execution>
+            <id>diktat</id>
+            <configuration>
+                <diktatConfigFile>/non/default/rule-set-file.yml</diktatConfigFile>
+            </configuration>
+        </execution>
+    </executions>
+    ```
+    </details>
+
+If you omit the `executionId`:
```

Comment:
Does it always use default default settings, even if custom configuration is tied to the same execution phase?

Reply:
The **default** `executionId` will be `default-cli`:

```console
$ mvn diktat:check
[INFO] Scanning for projects...
[INFO]
[INFO] ---------------< org.cqfn.diktat:diktat-examples-maven >----------------
[INFO] Building diktat-examples-maven 1.1.0-SNAPSHOT
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- diktat-maven-plugin:1.1.0:check (default-cli) @ diktat-examples-maven ---
[INFO] Running diKTat plugin with configuration file /home/alice/diktat-example-maven/foo.yml and inputs [/home/alice/diktat-example-maven/diktat-example
-maven/src/main/kotlin]
[WARNING] Not going to check indentation because there are tabs
[WARNING] You have enabled TRAILING_COMMA, but rule will remain inactive until you explicitly set configuration options. See [available-rules.md] for possible configuration options.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.467 s
[INFO] Finished at: 2022-05-26T19:05:49+03:00
[INFO] ------------------------------------------------------------------------
```

So, technically, yes, you can create a custom configuration with such a default name, but I doubt Maven has any kind of contract for this behaviour. I mean, we probably shouldn't rely on this, as it may change (and break).

Reply:
Looks like `default-cli` is part of official maven documentation: https://maven.apache.org/guides/mini/guide-default-execution-ids.html